operatorconfig/moduleconfig/application-mobility/v0.3.0/cert-crds.yaml

```diff
@@ -0,0 +1,1684 @@
+apiVersion: apiextensions.k8s.io/v1

```

Comment:
We can delete this file, right?

Reply:
Yes, this was left over from using separate crds for cert. Removing